src/schemas/Program/index.js

```diff
@@ -235,33 +242,6 @@ const typeDefs = gql`
 /* =========
 HTTP resolvers
  * ========= */
-const getIsoDate = (time) => (time ? new Date(parseInt(time) * 1000).toISOString() : null);
```

Comment:
See #694, many of these changes are from that branch
https://github.com/icgc-argo/platform-api/pull/694